### PR TITLE
Add automation to combine dependabot PRs

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -2,10 +2,6 @@
 daysUntilStale: 60
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 7
-# Issues with these labels will never be considered stale
-# exemptLabels:
- # - pinned
```
Comment (mzedel) [inline .github/stale.yml:7]: Would it make sense to apply the same `  - dependencies` exemption here as in `mender-cli`?

**Review by mzedel — APPROVED** [review]
Thanks for this! I wasn't sure about the non-backend repos considering the likely conflicts in the vendored dependencies - but we'll see soon how it works! 🤞

File: .github/stale.yml (hunk 1 above)
Comment (lluiscampos) [reply .github/stale.yml:7]: It is already there, line 15 :+1: I am just removing the "duplicate".

Comment (mzedel) [reply .github/stale.yml:7]: argh sorry for the noise, I just looked at the github diff 🙈

**Comment by lluiscampos** [comment]
> Thanks for this! I wasn't sure about the non-backend repos considering the likely conflicts in the vendored dependencies - but we'll see soon how it works! crossed_fingers

Yes. I am not 100% sure neither. This repository is the best candidate to try it out IMO :smiley:

**Pull request merged**